docs/source/io_formats/settings.rst

```diff
@@ -468,6 +468,24 @@ attributes/sub-elements:
 
     *Default*: None
 
+  :parameters:
+    If this attribute is given, it indicates that the source is to be
+    instantiated from an externally compiled source function, with parameters
+    defined by the string provided in this attribute. In this case, the
+    custom source library must define a class that inherits from the
+    ``openmc::CustomSource`` abstract class. This class must implement a
+    ``sample_source()`` function, which takes an array of integers as an
+    argument. The custom source library must also contain a ``create`` method,
+    which takes a serialized form of the source (as provided to the parameters
+    attribute) as an argument and returns a pointer to an instance of the custom
```

Comment:
Instead of saying "serialized form of the source", I would just be explicit and say "the string provided in this attribute".